Dockerfile

```diff
@@ -25,7 +25,7 @@ ENV PATH $PATH:/opt/vertica/bin
 
 WORKDIR /usr/src/app
 
-COPY README.md requirements.txt setup.py setuputils.py /usr/src/app/
```

Comment:
Whoops!  We got rid of this file in a previous PR, which broke building the Docker image here.